jekyll/_docs/api.md

```diff
@@ -219,7 +232,7 @@ The branch name should be url-encoded.
 {{ site.data.api.retry_build | api_endpoint }}
 
 You can retry a build with ssh by swapping "retry" with "ssh":
-`https://circleci.com/api/v1/project/:username/:project/:build_num/ssh`
+`https://circleci.com/api/v1.1/project/:vcs-type/:username/:project/:build_num/ssh`
```

Comment:
Nevermind, looks like [it's shipped](https://circleci.atlassian.net/browse/CIRCLE-45).